contracts/src/v0.8/VerifierProxy.sol

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
@austinborn is this ok?

Reply:
For testnet release this will be fine, but it will need review before mainnet launch. Asking whether this should block merging or not.

Reply:
Good to merge in the current versions with `UNLICENSED`.